src/DmvSimulator.java

```diff
@@ -1,11 +1,11 @@
 public class DmvSimulator {
 
     public static void main(String[] args) {
-        int spot = (int) Math.random()*100;
+        int spot = (int) Math.random()*200;
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Random value from 0 to 1 is coerced to the integer 0 (RV_01_TO_INT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-random-value-from-0-to-1-is-coerced-to-the-integer-0-rv-01-to-int).
A random value from 0 to 1 is being coerced to the integer value 0. You probably want to multiply the random value by something else before coercing it to an integer, or use the Random.nextInt(n) method.